### DISCUSSION TITLE
Straps For Battery

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#53] [General]
I have 2 straps printed to hold the battery to the bottom of the case.

One is relatively flat with very small protruding plastic at both ends on one side.

The other has bigger protrusions on either end on one side.

Which should be  used for the 3000mah LiPo battery?

Dave

**Comment by alex-code** [comment]
I could only design for the two sizes of battery I had.
My 3000mah was thinner so the flatter strap was used.

For a second throttle I made I could only get 2000mah batteries which were thicker so needed the other strap.
